src/ui/components/Carousel/index.js

```diff
@@ -64,7 +65,7 @@ export class CarouselBase extends React.Component {
           <NukaCarousel
             autoplay
             autoplayInterval={4000}
-            cellAlign={getDirection(i18n.lang) === 'ltr' ? 'left' : 'right'}
+            cellAlign={getDirection(i18n.lang) === LTR ? 'left' : 'right'}
```

Comment:
This file doesn’t exist anymore; you’ll want to update your code by pulling the newest master and rebasing on top of master. It’s why you have conflicts.